tests/vtestbed.csv

```diff
@@ -1 +0,0 @@
-../ansible/vtestbed.csv
```

Comment:
remove this file

Reply:
Oh, remove by mistake. Fixed now. Thanks